backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -108,45 +140,117 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
+                    }
+                }
+
+                project.TierLevel = GetTier(model.Market, project.Properties.Count()); // TODO: Need to import or link project properties.
                 project.TierLevelId = project.TierLevel.Id;
                 project.Risk = GetRisk(model.Risk);
                 project.RiskId = project.Risk.Id;
 
-                project.OcgFinancialStatement = model.OcgFinancialStatement;
+                project.Assessed = model.Assessed ?? project.Properties.Where(p => p.PropertyType == Entity.PropertyTypes.Land).Sum(p => p.Parcel.Evaluations.Where(e => e.Key == Entity.EvaluationKeys.Assessed).OrderByDescending(e => e.Date).FirstOrDefault()?.Value ?? 0);
                 project.NetBook = model.NetBook;
-                project.Estimated = model.Estimated;
-                project.ProgramCost = model.ProgramCost;
-                project.SalesCost = model.SalesCost;
-                project.InterestComponent = model.InterestComponent;
-                project.NetProceeds = model.NetProceeds;
-                project.GainLoss = model.GainLoss;
-                project.SaleWithLeaseInPlace = model.SaleWithLeaseInPlace;
-
-                if (model.PriorNetProceeds.HasValue)
-                {
-                    project.Snapshots.Add(new Entity.ProjectSnapshot(project) { NetProceeds = model.PriorNetProceeds.Value });
-                }
+                project.Market = model.Market;
+                project.Appraised = model.Appraised;
 
-                project.MarketedOn = model.MarketedOn;
+                project.SubmittedOn = new DateTime(model.ReportedFiscalYear - 1, 1, 1); // Don't have a source for this information.
+                project.ApprovedOn = project.SubmittedOn; // Don't have a source for this information.
                 project.CompletedOn = model.CompletedOn;
 
-                project.PrivateNote = model.PrivateNote;
+                var metadata = new Entity.Models.DisposalProjectSnapshotMetadata
+                {
+                    InitialNotificationSentOn = null, // Don't have a source for this information.
+                    ThirtyDayNotificationSentOn = null, // Don't have a source for this information.
+                    SixtyDayNotificationSentOn = null, // Don't have a source for this information.
+                    NinetyDayNotificationSentOn = null, // Don't have a source for this information.
+                    OnHoldNotificationSentOn = null, // Don't have a source for this information.
+                    InterestedReceivedOn = null, // Don't have a source for this information.
+                    TransferredWithinGreOn = null, // Don't have a source for this information.
+                    ClearanceNotificationSentOn = null, // Don't have a source for this information.
+                    RequestForSplReceivedOn = model.RequestForSplReceivedOn, // Don't have a source for this information.
```

Comment:
you say you don't have a source here, but it seems like you do?

Reply:
The source is the JSON file sent with the import.  But SIS doesn't have anything that matches.